test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -309,6 +317,22 @@ Describe "Get-Help should find pattern help files" -Tags "CI" {
         )
         $command.Invoke() | Should -Be $result
     }
+
+    It "Get-Help should fail expectedly searching for class help with hidden members" {
+        $testModule = @'
+        class foo
+        {
+            hidden static $monthNames = @('Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun','Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec')
+        }
+'@
+        $modulesFolder = Join-Path $TestDrive "Modules"
```

Comment:
Maybe use `-AdditionalChildPath` instead of two `Join-Path`

Reply:
`$modulesFolder` and `$modulePath` are used independently